ui/client/components/HealthCheck.js

```diff
@@ -1,40 +1,48 @@
-import React from "react";
-import {render} from "react-dom";
-import {connect} from "react-redux";
-import HttpService from "../http/HttpService";
+import React from "react"
+import {connect} from "react-redux"
+import HttpService from "../http/HttpService"
 import InlinedSvgs from '../assets/icons/InlinedSvgs'
 import PeriodicallyReloadingComponent from './PeriodicallyReloadingComponent'
+import ActionsUtils from "../actions/ActionsUtils";
 
 class HealthCheck extends PeriodicallyReloadingComponent {
 
   constructor(props) {
-    super(props);
+    super(props)
 
     this.state = {
       healthCheck: undefined,
-    };
+    }
+  }
+
+  getIntervalTime() {
+    let baseIntervalTime = _.get(this.props, "featuresSettings.intervalSettings.base", this.baseIntervalTime)
+    let processesIntervalTime = _.get(this.props, "featuresSettings.intervalSettings.healthCheck", this.intervalTime)
+    return processesIntervalTime || baseIntervalTime
   }
 
   reload() {
-    HttpService.fetchHealthCheck().then(
-      (check) => this.setState({ healthCheck: check })
-    );
+    HttpService.fetchHealthCheck().then((check) => this.setState({ healthCheck: check }))
   }
 
   render() {
-    const { healthCheck } = this.state;
+    const { healthCheck } = this.state
     if (!healthCheck || healthCheck.state === 'ok') {
-      return null;
+      return null
     }
 
     return (
       <div className="healthCheck">
         <div className="icon" title="Warning" dangerouslySetInnerHTML={{__html: InlinedSvgs.tipsWarning}} />
         <span className="errorText">{healthCheck.error || 'State unknown'}</span>
       </div>
-    );
-
+    )
   }
 }
 
-export default HealthCheck;
+
+const mapState = state => ({
+  featuresSettings: state.settings.featuresSettings,
```

Comment:
maybe let's add map only intervalSettings here? 
Here and in all places below?

Reply:
I think, this is okey. There is only reference to the object.

Reply:
yes, but it's a bit messy now as we pass all our configuration around...